cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -80,9 +81,9 @@
   private static final String TYPE_RUN_RECORD_STARTED = "runRecordStarted";
   private static final String TYPE_RUN_RECORD_SUSPENDED = "runRecordSuspended";
   private static final String TYPE_RUN_RECORD_COMPLETED = "runRecordCompleted";
+  private static final String TYPE_WORKFLOW_NODE_STATE = "workflowNodeState";
```

Comment:
should we start reducing the key size here? should not be necessary to store the entire string, right?